ui-testing/src/main/java/com/qreal/wmp/uitesting/dia/scene/Coordinate.java

```diff
@@ -0,0 +1,59 @@
+package com.qreal.wmp.uitesting.dia.scene;
+
+import com.codeborne.selenide.SelenideElement;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Optional;
+
+public class Coordinate {
+    
+    public static final String SELECTOR = "transform";
+    
+    public static final int POINT_IN_CELL = 25;
+    
+    private final int xAbsolute;
```

Comment:
By "5.2.7 Local variable names" of the Style guide we should use lowerCamelCase.